src/foam/dao/EasyDAO.js

```diff
@@ -514,6 +521,11 @@ return delegate;
       name: 'testData',
       generateJava: false
     },
+    {
+      name: 'serviceProviderAware',
+      class: 'Boolean',
+      value: false
```

Comment:
It would be slightly nicer in my opinion if this property had a factory that returns `foam.nanos.auth.ServiceProviderAware.class.isAssignableFrom(getOf().getObjClass())`. Then you would avoid a situation in which someone sets this property to false and therefore expects there not to be a ServiceProviderAwareDAO when in fact there might still be one added. It gives the option of explicitly setting this to false to ignore the fact that `of` might implement ServiceProviderAware.

Not crucial though, just a suggestion.

Reply:
I like that.